### PR TITLE
Add read me

### PR DESCRIPTION
**Pull request by ConfusedSatlan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,93 @@
...
+- [Contributing](#contributing)
+- [License](#license)
+
+## Getting Started
+
+### Prerequisites
+
+Before you begin, ensure you have the following prerequisites:
+
+- A working development environment with your preferred programming language and tools.
```
Comment (andrii-hoienko) [inline README.md]: what tools? what language?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,93 @@
...
+- Docker installed on your system for running the Bookstore API.
+
+### Installation
```
Comment (andrii-hoienko) [inline README.md:43]: Add alternative way to run app

File: README.md [hunk 3]
```diff
@@ -1,2 +1,93 @@
...
+- [Docker Setup](#docker-setup)
+- [.env Configuration](#env-configuration)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Getting Started
+
+### Prerequisites
+
+Before you begin, ensure you have the following prerequisites:
```
Comment (andrii-hoienko) [inline README.md]: Mention the specific functionalities of your controllers.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,93 @@
...
+The Bookstore API is containerized using Docker for ease of deployment. You can use the provided `docker-compose.yml` file to run the API and its dependencies, such as the PostgreSQL database.
+
+To start the Docker containers, run:
+
+```bash
+docker-compose up --build
+```
+## .env Configuration
+
+Configuration options for the Bookstore API are stored in the `.env` file. You can customize settings such as production and debugging ports by editing this file.
```
Comment (andrii-hoienko) [inline README.md:102]: Add something about Contribution

File: README.md [hunk 5]
```diff
@@ -1,2 +1,93 @@
...
+
+### Example .env file:
+
+```env
+# Info for DB
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=password
+POSTGRES_DATABASE=book-store
+POSTGRES_LOCAL_PORT=5434
+POSTGRES_DOCKER_PORT=5432
```
Comment (andrii-hoienko) [inline README.md:112]: Detail the technologies and tools used, like Spring Boot, Spring Security, Spring Data JPA, Swagger, etc.

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Please read and follow the hints

File: README.md [hunk 6]
```diff
@@ -1,2 +1,130 @@
...
+
+### Prerequisites
+
+Before you begin, make sure you have the following prerequisites in place:
+
+- **Development Environment:** Ensure that you have a working development environment set up for the Java programming language.
+
+- **Integrated Development Environment (IDE):** You should have a preferred IDE, such as IntelliJ IDEA, for coding, debugging, and project management.
+
+- **Docker:** Proficiency in using Docker for containerization to package and run your application.
```
Comment (vovasalyha) [inline README.md]: proficiency in Docker is not required for such a trivial setup. User should be able to run your app without deep knowledge of Docker simply using the commands from your guide

File: README.md [hunk 7]
```diff
@@ -1,2 +1,130 @@
...
+
+This project focuses on building a web API in Java using Spring Boot and containerizing it with Docker. The API's endpoints and functionality are documented using Swagger, and security measures are essential components of this web application.
+
+**Local Development Environment:** You can run your application on your local development machine without Docker. In this case, you might need to set up a local database, provide the necessary environment variables from the `.env` file, and run the application using your preferred development tools.
+
+### Installation
+
+1. Clone this repository to your local machine:
+
+   ```bash
```
Comment (vovasalyha) [inline README.md:47]: please, close the code blocks with tracing ```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,130 @@
...
+   git clone https://github.com/ConfusedSatlan/online-book-store.git
+Navigate to the project directory:
+
+bash
```
Comment (vovasalyha) [inline README.md]: make it a code block

File: README.md [hunk 9]
```diff
@@ -1,2 +1,130 @@
...
+Copy code
+cd bookstore-api
+Build and run the Docker containers:
+
+bash
```
Comment (vovasalyha) [inline README.md]: make it a code block

**Review by vovasalyha — CHANGES_REQUESTED** [review]
GJ, let's improve few more things

**Review by andrii-hoienko — APPROVED** [review]
GJ!

**Pull request merged**